grammar.js

```diff
@@ -709,7 +710,7 @@ module.exports = grammar({
         ),
 
         // Expressions
-        _expression: $ => choice(
+        _expression: $ => prec.right(choice(
```

Comment:
```suggestion
        _expression: $ => choice(
```

I think it is better if the associativity rule is only applied to calls